src/components/common/SessionAccounts.vue

```diff
@@ -14,40 +14,45 @@
     <div v-for="account in accounts" :key="account.address">
       <div class="card content">
         <div class="content-left">
-          <h3>{{ account.name }}</h3>
-          <Bech32 :address="account.address" short-form />
+          <h3>{{ account.key }}</h3>
+          <Bech32 :address="account.value" short-form />
         </div>
-        <TmBtn value="Go to Lunie" color="primary" />
       </div>
     </div>
+    <div class="button-add-account">
+      <TmBtn
+        value="Add Account"
+        color="primary"
+        @click.native="goTo('welcome')"
+      />
+    </div>
   </div>
 </template>
 
 <script>
-import TmBtn from "common/TmBtn"
 import ToolBar from "common/ToolBar"
+import TmBtn from "common/TmBtn"
 import Bech32 from "common/Bech32"
 export default {
-  name: `session-ext-accounts`,
+  name: `session-accounts`,
   components: {
-    TmBtn,
     ToolBar,
-    Bech32
+    Bech32,
+    TmBtn
+  },
+  props: {
+    goTo: {
+      type: Function,
+      required: true
+    }
   },
-  data: () => ({
-    accounts: [
-      {
-        address: "cosmos1ek9cd8ewgxg9w5x3benji0uf4aaxaruvcw4v9e",
-        name: "Account 1"
-      }
-    ]
-  }),
-  methods: {
-    setState(value) {
-      this.$emit(`route-change`, value)
-    },
-    close() {
-      this.$emit(`close`)
+  computed: {
+    accounts() {
+      let accounts = this.$store.state.accounts
+      return accounts.map(({ name, address }) => ({
+        value: address,
```

Comment:
any reason we're changing this from `address` to value: address`?

Reply:
Just keeping the code uniform as it was in TmSessionSignIn https://github.com/luniehq/lunie/blob/26a9e93b3012ce1c4cb94d24585a5787049cb9e9/src/components/common/TmSessionSignIn.vue#L88